app/helpers/orchid/display_helper.rb

```diff
@@ -22,7 +22,7 @@ def metadata(res, label, api_field, link: true, separator: " | ")
         end
       end
       html << dataArray
-                .map { |i| "<span>#{i}</span>" }
+                .map { |i| "#{i}" }
```

Comment:
I think changing this to
```ruby
.map { |i| "<dd>#{i}</dd>" |
```
is what we want to replace _all_ of the old `<li>` tags with `<dd>` tags

Reply:
fixed as suggested